README.md

```diff
@@ -135,13 +148,31 @@ When using as a github app the flow is the following:
 4. Event details are sent to your deployed bot instance (to a webhook URL that was set in github app's settings)
 5. The bot handles the event
 
+## Payments Permits in a local instance
+
+For payment to work in your local instance, ubiquibot must be set up in a Github organization. It will not work for a ubiquibot instance set up in a personal account. Once, you have an ubiquibot instance working in an organization, follow the steps given below:
+
+1. Create a new private repository in your Github organization with name `ubiquibot-config`
+2. Add your ubiquibot app to `ubiquibot-config` repository.
+3. Create a file `.github/ubiquibot-config.yml` in it. Fill the file with contents from [this file](https://github.com/ubiquity/ubiquibot/blob/development/.github/ubiquibot-config.yml).
+4. Go to https://pay.ubq.fi/keygen and generate X25519 public/private key pair. Fill private key of your wallet's address in `PLAIN_TEXT` field and click `Encrypt`.
+5. Copy the `CIPHER_TEXT` and append it to your repo `ubiquibot-config/.github/ubiquibot-config.yml` as
+
+   `private-key-encrypted: "PASTE_YOUR_CIPHER_TEXT_HERE"`
+
+6. Copy the `X25519_PRIVATE_KEY` and append it in your local ubiquibot repository `.env` file as
+
+   `X25519_PRIVATE_KEY=PASTE_YOUR_X25519_PRIVATE_KEY_HERE`
+
 ## How to QA any additions to the bot
 
-1. Fork the ubiquibot repo and install the [ubiquibot-qa app](https://github.com/apps/ubiquibot-qa) on the forked repository.
-2. Enable github action running on the forked repo and allow `issues` on the settings tab.
-3. Create a [QA issue](https://github.com/ubiquibot/staging/issues/21) similar to this where you show the feature working in the forked repo
-4. Describe carefully the steps taken to get the feature working, this way our team can easily verify
-5. Link that QA issue to the pull request as indicated on the template before requesting a review
+Make sure you have your local instance of ubiquibot running.
+
+1. Fork the ubiquibot repo and add your local instance of ubiquibot to the forked repository.
+2. Enable Github action running on the forked repo and allow `issues` on the settings tab.
```

Comment:
```suggestion
2. Go to repository settings and turn on `issues`.
```

Reply:
I see these actions inform the hunter about the wrong commit message format and lint problems before they create PR preventing the need for a forced push later. Sounds like a part of QA to me.

Reply:
Hmm okay I was just worried the bot will be triggered by actions but if the QA bot is not installed on the repo it will just fail, so it's fine